### ISSUE TITLE
Xterm Esc Seq CSI 11t returns window status 2 iconified instead of 1 open

### ISSUE THREAD

**Issue by BrianInglis** [mintty/mintty#893]
Looking for mintty window info I can get out of xterm esc seqs, I found that it returns 2 iconified instead of 1 open, according to https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h2-Miscellaneous as well as its CSI ... t section, and xterm docs, where 2 is also the window op to iconify and 1 is open: 
```
$ xterm_info
window position	^[[13t	3 px 210+121	^[[3;210;121t$
window size	^[[14;2t	4 px 892x881	^[[4;881;892t$
text position	^[[13;2t	3 px 219+153	^[[3;219;153t$
text size	^[[14t	4 px 840x840	^[[4;840;840t$
text chars	^[[18t	8 ch 120x60	^[[8;60;120t$
char size	^[[16t	6 px 7x14	^[[6;14;7t$
screen chars	^[[19t	9 ch 274x85	^[[9;85;274t$
screen size	^[[15t	5 px 1920x1200	^[[5;1200;1920t$
window status	^[[11t	2 icon	^[[2t$
```
where column 1 is the function, 2 is the esc seq, 3 is response prefix, 4 is value unit, 5 is response values, 6 is response esc seq cat -A output. 
Have not yet tried prompting, sleeping, iconifying, querying, sleeping, opening, querying for completeness.

**Comment by mintty** [comment]
Thanks, to be fixed easily.
While testing this, I'm facing weird obstacles; a test command like `echo -e '\e[11t'; read x` will also iconify mintty and I have no clue why that happens, it doesn't in xterm.
Another observation is that xterm reports CSI 1 t in Cygwin/X, whether iconified or not. It works on Linux.

**Comment by BrianInglis** [comment]
Not seeing those issues because I wrap tests in a shell function, issue the esc seq in the read prompt, delimiter set to expected end of response value, without echo, 1s timeout, e.g.:
```
$ t () { p=$'\E[11t'; read -p "$p" -d t -s -t 1 -N 80 r; cat -A <<< "$p"$'\t'"$r"; }
$ t
^[[11t^I^[[2t
```
Try wrapping in function, prompt to iconify, sleep 10, click iconify, query, echo result, sleep 10, wait 15s, click open, query, echo result, e.g.: 
```
$ t; echo iconify; sleep 10; t; sleep 10; t
^[[11t^I^[[2t$
iconify # prompt to you
^[[11t^I^[[1t$
^[[11t^I^[[2t$
```
that was using above function t(), running latest x64 and mintty.

**Comment by BrianInglis** [comment]
**Off Topic**: _doing Cygwin ITP of latest vttest 20190710 on cygwin-apps - please feel free to comment and state concerns, interests, issues on that list_

**Comment by mintty** [comment]
Released 3.0.3.
